lib/overhang.js

```diff
@@ -128,7 +131,10 @@ $.fn.overhang = function (arguments) {
     // Submit action
     $inputField.keydown(function (e) {
       if (e.keyCode == 13) {
-
+        if (e.ctrlKey) {
```

Comment:
Implementing a Shift + Enter to generate a new line is a more common used approach